### PR TITLE
Add net.redeclipse.RedEclipse

### PR DESCRIPTION
**Pull request by AsciiWolf**

### CODE REVIEW

**Comment by AsciiWolf** [comment]
bot, build net.redeclipse.RedEclipse

File: net.redeclipse.RedEclipse.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    "type": "file",
+                    "path": "redeclipse.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "redeclipse.sh"
+                }
+            ],
+            "build-commands": [
+                "cd src; cmake; make",
```
Comment (barthalion) [inline net.redeclipse.RedEclipse.json]: `no-make-install` + cmake-ninja buildsystem?

Comment (AsciiWolf) [reply net.redeclipse.RedEclipse.json]: Nope, I didn't contact upstream yet. I am however planning to do it.

RedEclipse already [has](https://github.com/red-eclipse/base/blob/master/src/install/nix/redeclipse.appdata.xml.am) an AppData file, but it is only a template that packagers can use, it is not in the Makefile and does not seem to be updated a lot. I have at least [submitted](https://github.com/red-eclipse/base/pull/824/commits/f81a76a66fe828ed43420cf3c4d6b8d278594a96) the OARS metadata, but I think it's better to use my own modified AppData file for now.

Same with the desktop file, it [exists](https://github.com/red-eclipse/base/blob/master/src/install/nix/redeclipse.desktop.am) as a template, but currently has an issue with the Keywords having bad separator which I [fixed](https://github.com/red-eclipse/base/pull/824/commits/cebec9e52877eebc17fd2e1d11ef9c1a047498aa) upstream, but it won't be fixed in the current version. I am also not sure, how to easily replace the "@APPNAME@" strings. So I am using my own desktop file for now.

Comment (AsciiWolf) [reply net.redeclipse.RedEclipse.json]: The reason why I used the _simple_ buildsystem is that the CMake files are not in the main directory, but in the "src" directory. Is there any way to specify the build directory?

Comment (AsciiWolf) [reply net.redeclipse.RedEclipse.json]: @jcaesar Thanks!

**Comment by AsciiWolf** [comment]
bot, build net.redeclipse.RedEclipse

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/net.redeclipse.RedEclipse

Thanks!

**Pull request closed without merging**